src/Sylius/Behat/Context/Ui/Admin/ManagingExchangeRatesContext.php

```diff
@@ -61,55 +63,55 @@ public function iWantToBrowseExchangeRatesOfTheStore()
      * @When /^I specify its ratio as (-?[0-9\.]+)$/
      * @When I don't specify its ratio
      */
-    public function iSpecifyItsRatioAs($ratio = null)
+    public function iSpecifyItsRatioAs(?string $ratio = null): void
     {
-        $this->createPage->specifyRatio($ratio ?? '');
+        $this->formElement->specifyField('Ratio', $ratio ?? '');
```

Comment:
I am not convinced that this is entirely a good change, I would be for using specific methods instead of this generic one, or at least to define the method that uses `getElement` underneath